sdk/src/clearingHouse.ts

```diff
@@ -1495,7 +1654,10 @@ export class ClearingHouse {
 	 * @param marketIndex
 	 * @returns
 	 */
-	public async closePosition(marketIndex: BN): Promise<TransactionSignature> {
+	public async closePosition(
```

Comment:
probably good to revert this?

Reply:
need the limit price field in the tests to put through large trades in a single ix 
```
async function fullClosePosition(clearingHouse, userPosition) {
	let limitPrice;
	if (userPosition.baseAssetAmount.lt(ZERO)) {
		limitPrice = new BN(100 * 1e13);
	} else {
		limitPrice = new BN(10 * 1e6);
	}

	return await clearingHouse.closePosition(new BN(0), limitPrice);
}
```
also, dont we need baa to be abs() ?